cypress/SETUP.md

```diff
@@ -16,11 +16,15 @@ In order to run the github e2e tests successfully, we need to perform the follow
 
 In order to run the email e2e tests successfully, we need to perform the following one-off setup:
 
-1. go to the **production** database (reader instance)
-2. search for the `e2e email test site` in `sites`
-3. copy the value over to your local db in `sites`
-4. search for the `e2e-email-test-repo` in `repos`
-5. copy the value over to your local db in `repos`
+1. go to the local database
+2. Run these commands:
+
+```
+INSERT INTO sites (name, site_status, job_status, creator_id)
+VALUES ('e2e email test site', 'INITIALIZED', 'READY', '1');
```

Comment:
the `creator_id` in `sites` table is integer type (`int8`) instead of text/string, we should use `1` without the quote here to match the data type.

Reply:
sure, but just a heads up that [postgres auto converts the type for us](https://www.postgresql.org/docs/current/typeconv.html)

Reply:
Understand that in this case since text `1` can be parsed to number `1` without any ambiguity, postgres will perform the implicit conversion. But still the best practice is to use the correct type that matches the target column when it's possible.